### PR TITLE
ENG-2284 #comment Create TagsInput component

### PR DESCRIPTION
**Pull request by patricia-mps**

### CODE REVIEW

File: src/elements/inputGroup/InputGroup.tsx [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+  placeholder,
   disabled,
   error,
+  size = 'lg',
+  onChange,
+  onFocus,
+  onBlur,
 }: Props): JSX.Element => (
-  <div>
+  <div className={classnames(`size-${size}`, 'form-group')}>
```
Comment (pedroguia) [inline src/elements/inputGroup/InputGroup.tsx:24]: The prop "className" that the component receives is usually passed to the first div of the component, so we can customize the style of the entire component and not only the style of the input. Can you move it up here?

File: src/elements/inputTags/InputTags.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        return;
+      }
+      onChange && onChange([...tags, text]);
+      e.target.value = '';
+      e.preventDefault();
+    }
+  };
+
+  return (
+    <div className={classnames(`size-${size}`, 'form-group')}>
```
Comment (pedroguia) [inline src/elements/inputTags/InputTags.tsx:58]: The prop "className" that the component receives is usually passed to the first div of the component, so we can customize the style of the entire component and not only the style of the input. Can you move it up here?

File: src/elements/inputTags/InputTags.types.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  dataTestId?: string;
+  placeholder?: string;
+  helperText?: string;
+  disabled?: boolean;
+  seprators?: Array<string>;
+  maxTags?: number;
+  minLength?: number;
+  maxLength?: number;
+  size?: 'sm' | 'md' | 'lg';
+  inputLabelProps?: Label;
```
Comment (pedroguia) [inline src/elements/inputTags/InputTags.types.ts:28]: This "inputLabelProps" should have its type imported from the interface of the "InputLabel" component. Can you create a file name "InputLabel.types.ts" with its interface in the component folder, and import that interface in every inputLabelProps prop across the toolkit?

File: src/elements/inputGroup/InputGroup.types.ts [hunk 4]
```diff
@@ -1,9 +1,19 @@
 interface Props {
+  value: string;
+  id?: string;
+  className?: string;
+  inputLabelProps?: any;
```
Comment (pedroguia) [inline src/elements/inputGroup/InputGroup.types.ts:5]: This "inputLabelProps" should have its type imported from the interface of the "InputLabel" component. Can you create a file name "InputLabel.types.ts" with its interface in the component folder, and import that interface in every inputLabelProps prop across the toolkit?

File: src/elements/inputGroup/InputGroup.tsx [hunk 5]
```diff
@@ -19,12 +31,23 @@ const InputGroup: FC<Props> = ({
       )}
       <input
         type="text"
-        className={classnames('form-control', {
-          borderRadiusLeft: !prepend,
-          borderRadiusRight: !append,
-        })}
-        placeholder={inputPlaceholder}
+        value={value}
+        id={id}
```
Comment (pedroguia) [inline src/elements/inputGroup/InputGroup.tsx:35]: We also need a prop "name" for this input please.

**Review by pedroguia — CHANGES_REQUESTED** [review]

File: src/elements/inputGroup/InputGroup.tsx (hunk 1 above)
Comment (patricia-mps) [reply src/elements/inputGroup/InputGroup.tsx]: Yes, you are right. Done :+1:

File: src/elements/inputTags/InputTags.tsx (hunk 2 above)
Comment (patricia-mps) [reply src/elements/inputTags/InputTags.tsx]: Yes, you are right. Done 👍

File: src/elements/inputGroup/InputGroup.types.ts (hunk 4 above)
Comment (patricia-mps) [reply src/elements/inputGroup/InputGroup.types.ts]: Done 👍

File: src/elements/inputGroup/InputGroup.tsx (hunk 5 above)
Comment (patricia-mps) [reply src/elements/inputGroup/InputGroup.tsx:37]: Done 👍

File: src/elements/inputTags/InputTags.types.ts (hunk 3 above)
Comment (patricia-mps) [reply src/elements/inputTags/InputTags.types.ts]: Done 👍

File: src/elements/forms/inputTags/InputTags.tsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+  return (
+    <InputTagsComponent
+      onChange={e => {
+        handleChange(e);
+        if (onChange) onChange(e);
+      }}
+      tags={value}
+      name={fieldName}
+      {...inputTagsFormProps}
```
Comment (pedroguia) [inline src/elements/forms/inputTags/InputTags.tsx:25]: inputTagsFormProps => inputTagsProps

**Review by pedroguia — CHANGES_REQUESTED** [review]

File: src/elements/forms/inputGroup/InputGroup.tsx [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+  return (
+    <InputGroupComponent
+      onChange={e => {
+        handleChange(e);
+        if (onChange) onChange(e);
+      }}
+      value={value}
+      name={fieldName}
+      error={error ? error.message : undefined}
+      {...inputGroupFormProps}
```
[the remark this answers is not in the document]
Comment (patricia-mps) [reply src/elements/forms/inputGroup/InputGroup.tsx]: Done 👍

File: src/elements/forms/inputTags/InputTags.tsx (hunk 6 above)
Comment (patricia-mps) [reply src/elements/forms/inputTags/InputTags.tsx]: Done 👍

**Review by pedroguia — APPROVED** [review]

**Review by itsnunolemos — APPROVED** [review]

**Pull request merged**